lib/diagnostic.js

```diff
@@ -12,15 +12,29 @@
 //
 
 'use strict';
-
 const fs = require('fs');
-
 let inFile = process.argv[2];
 
+
+fs.readFile(inFile, (error, file) => {
+  if (error) throw error;
+
+  let lines = file.split('\n');
+  lines.pop();
+  let sumLines = lines.reduce(function(a, b){ return a + b; });
+  return sumLines;
+
+});
+
```

Comment:
This code will not work as written because the inputs will be read as strings - you need to convert them to Numbers before they can be added.